Java/TelemetrySystem/src/main/java/tddmicroexercises/telemetrysystem/ClientClass.java

```diff
@@ -0,0 +1,75 @@
+package tddmicroexercises.telemetrysystem;
+
+import java.util.Random;
+
+import static tddmicroexercises.telemetrysystem.TelemetryClient.DIAGNOSTIC_MESSAGE;
+
+public class ClientClass implements Client{
```

Comment:
Lets not prefix or suffix the class name with "Class"
Rename it

Reply:
Renamed the classes